### ISSUE TITLE
Canonical transcript mostly of adjacent gene

### ISSUE THREAD

**Issue by klaricch** [konradjk/exac_browser#274]
Exac user: I am concerned about KCTD7 (http://exac.broadinstitute.org/gene/ENSG00000243335). The transcript stated in your header is ENST00000510829, which appears to be a transcript for the human paralog RABGEF1. Is it expected to have transcripts from paralogs mapped to each other in ExAC? Am I misinterpreting something?

Issue: The genomic coordinates for KCTD7 spans 7:66093868-66276446 on ensembl:
http://grch37.ensembl.org/Homo_sapiens/Gene/Summary?g=ENSG00000243335;r=7:66093868-66276446

This region includes both KCTD7 and RABGEF1 7:66093868-66276446 (http://genome.ucsc.edu/cgi-bin/hgTracks?db=hg19&lastVirtModeType=default&lastVirtModeExtraState=&virtModeType=default&virtMode=0&nonVirtPosition=&position=chr7%3A66093868%2D66276446&hgsid=507718049_pi0uPGo0wzClXxlfzNvx3skgf66e)

Unfortunately, in selecting a canonical transcript for KCTD7 the browser has selected a canonical transcript that spans the two adjacent genes which just happen to be paralogous.

**Comment by konradjk** [comment]
Yep, looks like a Gencode v19 annotation issue that's been resolved in future releases (GRCh38, but for that matter Gencode 24liftover37). I think we should keep it as is for the v1 version of the browser and consider a jump up to 24 for v2.
